### PR TITLE
feat(servicecontrol): generate library

### LINKED ISSUE #8082: Service Control API: `servicecontrol.googleapis.com`

### PR DESCRIPTION
**Pull request by coryan**
Fixes #8082

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/8094)

### CODE REVIEW

File: google/cloud/servicecontrol/README.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 :construction:
 
 This directory contains an idiomatic C++ client library for the
-[Service Control API][cloud-service-docs], a service to Provides admission control and telemetry reporting for services integrated with Service Infrastructure.
+[Service Control API][cloud-service-docs]. Provides admission control and
```
Comment (dbolduc) [inline google/cloud/servicecontrol/README.md:6]: optionally, we could use complete sentences:
s/Provides/This service provides/

File: google/cloud/servicecontrol/doc/main.dox [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 @mainpage Service Control API C++ Client Library
 
-An idiomatic C++ client library for the [Service Control API][cloud-service-docs], a service
-to Provides admission control and telemetry reporting for services integrated with Service Infrastructure.
+An idiomatic C++ client library for the [Service Control API][cloud-service-docs].
+Provides admission control and telemetry reporting for services integrated with
```
Comment (dbolduc) [inline google/cloud/servicecontrol/doc/main.dox:6]: same

**Review by dbolduc — APPROVED** [review]

File: google/cloud/servicecontrol/README.md (hunk 1 above)
Comment (coryan) [reply google/cloud/servicecontrol/README.md]: Done.

File: google/cloud/servicecontrol/doc/main.dox (hunk 2 above)
Comment (coryan) [reply google/cloud/servicecontrol/doc/main.dox]: Ditto.

**Pull request merged**